### PR TITLE
dont touch links if a relative link to another doc is not found

### PR DESCRIPTION
**Pull request by jckr**

### CODE REVIEW

File: modules/gatsby-theme-ocular/src/templates/doc-page-markdown.jsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ const CustomLinkWrapper = relativeLinks => {
       return <A href={href} {...props} />;
     }
     const hrefWithoutLeadingSlash = href.startsWith('/') ? href.slice(1) : href;
-    return <A {...props} href={relativeLinks[hrefWithoutLeadingSlash]} />;
+    return <A {...props} href={relativeLinks[hrefWithoutLeadingSlash]} || href/>;
```
Comment (ibgreen) [inline modules/gatsby-theme-ocular/src/templates/doc-page-markdown.jsx]: The fix is fine, however from a maintainability perspective, stashing behavior that requires a PR to fix inside an inline expression without comments is pretty subtle and not very self-documenting.

How about a helper function 

```js
function resolveLink(href) {
   const hrefWithoutLeadingSlash = href.startsWith('/') ? href.slice(1) : href;
   // dont touch links if a relative link to another doc is not found
   return relativeLinks[hrefWithoutLeadingSlash]} || href;
}
```

even better:
-  start extracting that function to a helper file `parse-links.js` or similar.

**Comment by jckr** [comment]
@ibgreen I have put all the links util functions (on the doc creation side and the ones we refer to in the first version of that PR) into the same util file as suggested.

**Comment by ibgreen** [comment]
>  have put all the links util functions (on the doc creation side and the ones we refer to in the first version of that PR) into the same util file as suggested.

\o/ This is excellent! It will make it so much easier for contributors since it separates the link processing from the gatsby page generation and the react rendering!

**Comment by jckr** [comment]
@ibgreen what's the process for versioning? I see there are several alpha versions of 1.1.0 on npm

**Comment by ibgreen** [comment]
@jckr Yes, this is a good time to go to official 1.1.0. 

Keep the change log / what's new / docs updated?

**Pull request merged**